dds/DCPS/DataReaderImpl.h

```diff
@@ -566,7 +566,10 @@ class OpenDDS_Dcps_Export DataReaderImpl
 
   virtual ICE::Endpoint* get_ice_endpoint();
 
-protected:
+  // TODO(sonndinh): public method?
```

Comment:
No I don't think it needs to be public.

Reply:
I changed it to protected. `setup_serialization` in `DataWriterImpl_T.h` is currently public. Maybe we should also change it to protected or private if it's not going to be called from outside?

Reply:
Sure that sounds fine. Also change the pure virtual in DataWriterImpl.h, if it's public.

Reply:
Actually it is called by some tests. So I'm gonna leave it as public.

Reply:
Ah, must have I forgot about that.